check_process

```diff
@@ -13,7 +13,7 @@
         setup_private=0
         setup_public=0
 		upgrade=1
-        upgrade=1	from_commit=b89554865459520d7d90a5a666f09f6987f41de6
+        #upgrade=1	from_commit=b89554865459520d7d90a5a666f09f6987f41de6
```

Comment:
Why removing test upgrade from previous version?

Reply:
Test upgrade from previous version was freezing the CI... putting it back...

Reply:
At least you can test upgrade from current version 0b1c3be0716e4800f2ed6f5b3065f707ed67b6bf